### ISSUE TITLE
Recompile for Apple Silicon

### ISSUE THREAD

**Issue by Coder84619** [nicollasricas/streamdeckvsc#15]
Please recompile the plugin to natively support Apple Silicon.

**Comment by romanr** [comment]
Yes

**Comment by strejda603** [comment]
Yes, please! It used to be amazing plugin, which could save a LOT of work... Now it's just useless, because it can't stay connected at all on Silicon Macs.. Please fix this!
